docs/UserGuide.md

```diff
@@ -85,7 +85,7 @@ e.g `(n/NAME)/(p/PHONE)` can be used as `n/John Doe` or as `p/12341234`
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `clients`, `exit` and `clear`) will be ignored.<br>
```

Comment:
Correct me if I'm wrong, but I think `list` was changed to `listContacts` and the command to switch to the view of these contacts is `contacts`